cylc/flow/job_file.py

```diff
@@ -201,14 +198,7 @@ def _write_suite_environment(self, handle, job_conf, run_d):
 
         handle.write('\n')
         # override and write task-host-specific suite variables
-        work_d = get_remote_suite_work_dir(
-            job_conf["platform"], job_conf['suite_name'])
         handle.write('\n    export CYLC_SUITE_RUN_DIR="%s"' % run_d)
-        if work_d != run_d:
-            # Note: not an environment variable, but used by job.sh
-            handle.write(
-                '\n    export CYLC_SUITE_WORK_DIR_ROOT="%s"' % work_d
-            )
```

Comment:
This is only set in one place now (nice) so can remove the Bash var substitution in `job.sh`:

```diff
-CYLC_SUITE_WORK_DIR_ROOT="${CYLC_SUITE_WORK_DIR_ROOT:-${CYLC_SUITE_RUN_DIR}}"
+CYLC_SUITE_WORK_DIR_ROOT="${CYLC_SUITE_RUN_DIR}"
```

Reply:
Seeing as that var is not exported, I've removed it altogether